WcaOnRails/app/helpers/results_helper.rb

```diff
@@ -30,7 +30,7 @@ def historical_pb_markers(results)
   end
 
   def link_to_ranking(event_id, type, &block)
-    url = "/results/events.php?eventId=#{event_id}&#{type}=true"
+    url = "/results/rankings/#{event_id}/#{type}"
```

Comment:
`results_rankings_path(event_id, type)` doesn't seem to work, it's giving URLs like `results/rankings.event_id`. I've read that this is because you can't pass parameters like this to the `index` action. Maybe I could rename the action to `rankings` and try?

Another option is using `results_rankings_path(event_id: event_id, type: type)`, but it produces URLs like `/results/rankings?event_id=event_id&type=type`, which don't work either :slightly_frowning_face: 

The last option I can think of is leaving the helper (but as as an application helper) and use it for all the links to rankings.

Reply:
Funnily enough, `results_rankings_path(event_id, type)` works perfectly with the changes from 07e556770fe1a23609148917668b78f93a6ec149 to be able to test this out on staging :confused: